scripts/format_solidity.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+CHECK=
+HELP=
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -c | --check) CHECK=true ;;
+        -h | --help) HELP=true ;;
+    esac
+    shift
+done
+
+if [ "$HELP" = true ]; then
+    echo "Usage: ./scripts/lint.sh [OPTIONS]"
```

Comment:
```suggestion
    echo "Usage: ./scripts/format_solidity.sh [OPTIONS]"
```

Actually, any reason to not combine this with existing `lint.sh` script? I don't feel strongly either way, but generally prefer to have fewer scripts to reduce the amount of steps that need to be taken in typical flows.